Sorry: TabError: inconsistent use of tabs and spaces in indentation (gps_publisher.py, line 34)
